kubernetes/samples/scripts/create-weblogic-domain-pv-pvc/README.md

```diff
@@ -0,0 +1,227 @@
+
+```
+  ./create-pv-pvc.sh \
+  -i create-pv-pvc-inputs.yaml \
+  -o /path/to/output-directory
+```
+
+By default, the script generates two yaml files, namely `weblogic-sample-pv.yaml` and `weblogic-sample-pvc.yaml`, in the `/path/to/output-directory` directory. These two yaml files can be used to create the Kubernetes resources using the `kubectl create -f` command.
+
+```
+  kubectl create -f weblogic-sample-pv.yaml
+  kubectl create -f weblogic-sample-pvc.yaml
+
+```
+
+As a convenience, the script can optionally create the PV and PVC resources as well using the `-e` option.
+
+The usage of the create script is as follows.
+
+```
+$ sh create-pv-pvc.sh -h
+usage: create-pv-pvc.sh -i file -o dir [-e] [-h]
+  -i Parameter inputs file, must be specified.
+  -o Output directory for the generated yaml files, must be specified.
+  -e Also create the Kubernetes objects using the generated yaml files
+  -h Help
+```
+
+If you copy the sample scripts to a different location, make sure that you copy everything in the `<weblogic-kubernetes-operator-project>/kubernetes/samples/scripts` directory togather into the target directory, maintaining the orignal directory heirachy.
+
+
+## Configuration parameters 
+
+The following parameters in the inputs file can be customized if needed.
+
+| Parameter | Definition | Default |
+| --- | --- | --- |
+| `domainUID` | ID of the domain resource that the generated PV and PVC will be dedicated to. Leave it empty if the PV and PVC are going to be shared by multiple domains. | no default |
+| `namespace` | Kubernetes namespace to create the PVC. | `default` |
+| `baseName` | Base name of the PV and PVC. The generated PV and PVC will be <baseName>-pv and <baseName>-pvc respectively | `weblogic-sample` |
+| `weblogicDomainStoragePath` | Physical path of the storage for the PV. | no default |
+| `weblogicDomainStorageReclaimPolicy` | Kubernetes PVC policy for the persistent storage. The valid values are: 'Retain', 'Delete', and 'Recycle' | `Retain` |
+| `weblogicDomainStorageSize` | Total storage allocated for the PVC. | `10Gi` |
+| `weblogicDomainStorageType` | Type of storage. Legal values are `NFS` and `HOST_PATH`. If using 'NFS', weblogicDomainStorageNFSServer must be specified | `HOST_PATH` |
+| `weblogicDomainStorageNFSServer`| Name of the IP address of the NFS server. This setting only applies if weblogicDomainStorateType is NFS  | no default |
+
+By default, the domainUID is left empty in the inputs file so that the generated PV and PVC can be shared by multiple domain resources. For the use cases where dedicated PV and PVC are desired for a particular domain, the domainUID can be set, which will cause the generated PV and PVC associated with the domainUID specified. In the per domain PV and PVC case, the names of the generated yaml files and the Kubernetes PV and PVC objects are all decorated with the `domainUID`, and the PV and PVC obects are also labeled with the `domainUID`.
```

Comment:
"..., which will cause the generated PV and PVC associated with the domainUID (to be?) specified."